extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/internal/SonosHandlerFactory.java

```diff
@@ -29,47 +29,44 @@
 
 import com.google.common.collect.Lists;
 
-
 /**
- * The {@link SonosHandlerFactory} is responsible for creating things and thing 
+ * The {@link SonosHandlerFactory} is responsible for creating things and thing
  * handlers.
  * 
  * @author Karel Goderis - Initial contribution
  */
 public class SonosHandlerFactory extends BaseThingHandlerFactory {
-	
-	private Logger logger = LoggerFactory.getLogger(SonosHandlerFactory.class);
-    
-	private UpnpIOService upnpIOService;
-	private DiscoveryServiceRegistry discoveryServiceRegistry;
-
-	// optional OPML URL that can be configured through configuration admin 
-	private String opmlUrl = null;
-	
-    private final static Collection<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Lists.newArrayList(ZONEPLAYER_THING_TYPE_UID);
-    
+
+    private Logger logger = LoggerFactory.getLogger(SonosHandlerFactory.class);
+
+    private UpnpIOService upnpIOService;
+    private DiscoveryServiceRegistry discoveryServiceRegistry;
+
+    // optional OPML URL that can be configured through configuration admin
+    private String opmlUrl = null;
+
     protected void activate(ComponentContext componentContext) {
-    	super.activate(componentContext);
-    	Dictionary<String, Object> properties = componentContext.getProperties();
-		opmlUrl = (String) properties.get("opmlUrl");
+        super.activate(componentContext);
+        Dictionary<String, Object> properties = componentContext.getProperties();
+        opmlUrl = (String) properties.get("opmlUrl");
     };
-    
+
     @Override
-    public Thing createThing(ThingTypeUID thingTypeUID, Configuration configuration,
-            ThingUID thingUID, ThingUID bridgeUID) {
+    public Thing createThing(ThingTypeUID thingTypeUID, Configuration configuration, ThingUID thingUID,
+            ThingUID bridgeUID) {
 
-        if (ZONEPLAYER_THING_TYPE_UID.equals(thingTypeUID)) {
-            ThingUID sonosPlayerUID = getPlayerUID(thingTypeUID, thingUID, configuration);
-            logger.debug("Creating a sonos zone player thing with ID '{}'",sonosPlayerUID);
-            return super.createThing(thingTypeUID, configuration, sonosPlayerUID,null);
+        if (SUPPORTED_THING_TYPES_UIDS.contains(thingTypeUID)) {
```

Comment:
The SUPPORTED_THING_TYPES_UIDS contains the ZONEPLAYER_THING_TYPE_UID as referenced from SonosBindingConstants from now on.